src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+function getMarried({ isMarried, sex, partnerName }) {
+  if (!isMarried) {
+    return 'I am not married';
+  }
+
+  if (sex === 'm') {
+    return `${partnerName} is my wife`;
+  }
+
+  return `${partnerName} is my husband`;
```

Comment:
Better create a variable for wife/husband, so you can use ternary operator with isMarried instead of `if`s